### PR TITLE
Add new qassets to Quicksilver and Osmosis

### PR DESCRIPTION
**Pull request by joe-bowman**
Add qJUNO, qDYDX, qSAGA and qBLD to Osmosis and Quicksilver asset lists
Update qAsset images (png and SVG) for all qassets

### CODE REVIEW

File: quicksilver/assetlist.json [hunk 1]
```diff
@@ -228,6 +228,162 @@
...
+        },
+        {
+          "denom": "qbld",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uqbld",
+      "name": "Quicksilver Liquid Staked BLD",
+      "display": "qblud",
```
Comment (joe-bowman) [inline quicksilver/assetlist.json:365]:
```suggestion
      "display": "qbld",
```

File: osmosis/assetlist.json [hunk 2]
```diff
@@ -16290,8 +16290,224 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
       },
       "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/ashLAB.png"
+        }
+      ]
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis/assetlist.json:16298]: all the other images have no padding, but the qatom images have a bit of padding. Can you please provide an image without padding so the sizing on displays will be consistent?

File: osmosis/assetlist.json [hunk 3]
```diff
@@ -16538,29 +16538,218 @@
       ]
     },
     {
-      "description": "Beer Is Good for You!",
```
Comment (joe-bowman) [inline osmosis/assetlist.json:16541]: This looks ugly, but the BWH asset is listed twice in master. If you check individual commits, it looks cleaner :)

File: osmosis/assetlist.json (hunk 2 above)
Comment (joe-bowman) [reply osmosis/assetlist.json]: Done @JeremyParish69

**Pull request merged**